cpp/mcap/include/mcap/writer.hpp

```diff
@@ -79,7 +79,7 @@ struct MCAP_PUBLIC McapWriterOptions {
   bool forceCompression = false;
   /**
    * @brief The recording profile. See
-   * <https://github.com/foxglove/mcap/tree/main/docs/specification/profiles>
+   * <https://github.com/foxglove/mcap/blob/main/website/docs/spec/registry.md#profiles>
```

Comment:
Maybe better to link to the site version: https://mcap.dev/spec/registry#well-known-profiles